junit-plugin/src/main/scala/org/scalajs/junit/plugin/ScalaJSJUnitPlugin.scala

```diff
@@ -108,7 +108,7 @@ class ScalaJSJUnitPlugin(val global: Global) extends NscPlugin {
     import global._
 
     val phaseName: String = "junit-inject"
-    val runsAfter: List[String] = List("mixin")
+    val runsAfter: List[String] = List("mixin", "junit-make-mixin-bridges")
```

Comment:
That's really surprising, because it worked for me on master.
Did you clear your ivy cache of the old, buggy mixin plugin?